### PR TITLE
Move from Assurance_publish branch to staging

### PR DESCRIPTION
**Pull request by PravinPK**

### CODE REVIEW

File: packages/assurance/README.md [hunk 1]
```diff
@@ -6,6 +6,11 @@
 
 `@adobe/react-native-aepassurance` is a wrapper around the iOS and Android [AEPAssurance SDK](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-experience-platform-assurance) to allow for integration with React Native applications. Functionality to start Assurance session is provided through JavaScript documented below.
 
+## Prerequisites
+
+The Adobe Experience Platform Identity for Edge Network extension has the following peer dependency, which must be installed prior to installing the identity extension:
```
Comment (cacheung) [inline packages/assurance/README.md:11]: This is for Edge Identity, do you want to say Assurance?

Comment (PravinPK) [reply packages/assurance/README.md:11]: copy pasta.. Yes I will change it.
Thank you!

File: packages/assurance/README.md [hunk 2]
```diff
@@ -6,6 +6,11 @@
 
 `@adobe/react-native-aepassurance` is a wrapper around the iOS and Android [AEPAssurance SDK](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-experience-platform-assurance) to allow for integration with React Native applications. Functionality to start Assurance session is provided through JavaScript documented below.
```
Comment (cacheung) [inline packages/assurance/README.md:7]: Use [Adobe Experience Platform Assurance] instead of [AEPAssurance SDK]

**Review by cacheung — COMMENTED** [review]
In packages/assurance/README.md 
1. Under "Install npm package"
Install the @adobe/react-native-aepassurance alpha package:
(let remove alpha)

2. Can we remove line 76 Identity.registerExtension();  in Android setup?
So user won't get  confused how to register EdgeIdentity or core Identity through this example.  It is covered in EdgeIndentity ReadMe.

File: packages/assurance/README.md [hunk 3]
```diff
@@ -1,31 +1,28 @@
...
-npm install @adobe/react-native-aepassurance@3.0.0-alpha.1
+npm install @adobe/react-native-aepassurance
 ```
 
-For iOS development, after installing the plugins from npm, download the pod dependencies by running the following command:
-`cd ios && pod install && cd ..`
-To update native dependencies to latest available versions, run the following command:
-`cd ios && pod update && cd ..`
-
 ### Link
```
Comment (cacheung) [inline packages/assurance/README.md:27]: Do we need the Link Session here? Or we can point to the core Readme.
Can we do the similar way as other extension
Initialize the SDK via native code inside your AppDelegate and MainApplication in iOS and Android respectively, documentation on how to initialize the SDK can be found [here](https://github.com/adobe/aepsdk-react-native#initializing).  So we make the connection for users to go to Core Readme.

We can still keep the Example of registering Assurance code here.

**Comment by yangyansong-adbe** [comment]
@PravinPK  can you also update the Assurance released information on main README, like this section https://github.com/adobe/aepsdk-react-native/tree/staging#about-this-project?

**Review by cacheung — APPROVED** [review]
Looks good!

**Review by yangyansong-adbe — APPROVED** [review]

**Pull request merged**